Directory.Build.props

```diff
@@ -266,8 +266,13 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <!-- Embed IBC data on Windows release builds, if IBCMerge isn't available this will just log the commandline -->
-    <EnablePartialNgenOptimization Condition="'$(EnablePartialNgenOptimization)' == '' and '$(OS)' == 'Windows_NT' and '$(ConfigurationGroup)' == 'Release'">true</EnablePartialNgenOptimization>
+    <!-- Embed IBC data on release builds, if IBCMerge isn't available this will just log the commandline -->
+    <EnablePartialNgenOptimization Condition="'$(EnablePartialNgenOptimization)' == '' and '$(ConfigurationGroup)' == 'Release'">true</EnablePartialNgenOptimization>
```

Comment:
@MichalStrehovsky, were you also planning to switch this to full NGEN and then mark individual assemblies as partial in core-setup?

Reply:
It's doing full NGen since #36450. EnablePartialNGenOptimization wasn't doing anything since dotnet/arcade#2198 (the way it evaluates EnablePartialNGenOptimization makes that a no-op in CoreFX). I'll rename this property a subsequent pull request.